bootstrap/autoload.php

```diff
@@ -1,60 +1,72 @@
 <?php
 
-/*----------------------------------------------------*/
-// Paths
-/*----------------------------------------------------*/
 $rootPath = dirname(__DIR__);
-$webrootPath = $rootPath.DS.'htdocs';
 
 /*----------------------------------------------------*/
 // Composer autoload
 /*----------------------------------------------------*/
 if (file_exists($autoload = $rootPath.DS.'vendor'.DS.'autoload.php')) {
-    require_once $autoload;
+    die("Couldn't load the Composer autoload file.");
 }
+require_once $autoload;
```

Comment:
If autoload exists, die, otherwise require it?
I assume you mean if (!file_exists....)